frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte

```diff
@@ -91,11 +91,13 @@
 				report
 			/>
 		</div>
-		<h5 class="mt-4 ml-1 text-lg">Author: {data.report.ownerName}</h5>
-		<hr class="mt-4 text-grey-light" />
+		<h5 class="mt-4 text-lg">Author: {data.report.ownerName}</h5>
+		<span class="text-grey-darker mt-2">{new Date(data.report.updatedAt).toLocaleDateString()}</span
```

Comment:
I would add `Updated` before the date here. no colon needed just `Update 10/10/2023 at 12:23`

Reply:
Actually, maybe fine with time generally:
<img width="1003" alt="image" src="https://github.com/zeno-ml/zeno-hub/assets/5690524/4d03804e-fb60-45da-8dc0-fc971680d3fc">